kubedriver/kegd/action_handlers/deploy_helm_handler.py

```diff
@@ -43,10 +43,10 @@ def handle(self, action, parent_task_settings, script_name, keg_name, keg_status
             tmp_dir, chart_path, values_path = self.__write_chart(action)
             captured_objects = []
             if action_type == 'Install':
-                helm_client.install(chart_path, action.name, action.namespace, values=values_path)
+                helm_client.install(chart_path, action.name, helm_status.namespace, values=values_path)
```

Comment:
Although they are the same value can we replace `helm_status.namespace` with `action.namespace`? 
Or replace `action.name` with `helm_status.name` - just so it's consistent